mailbox/maildir/src/main/java/org/apache/james/mailbox/maildir/mail/model/MaildirMessage.java

```diff
@@ -236,11 +235,6 @@ public Long getTextualLineCount() {
     }
 
     @Override
-    public MessageId getMessageId() {
```

Comment:
I don't understand this change.
A message should be addressable by a unique id, regardless of the containing mailbox.
Putting the id into MailboxMessage looks like the opposite of this aim.
Am I missing something ?